var/spack/repos/builtin/packages/py-gensim/package.py

```diff
@@ -16,18 +16,27 @@ class PyGensim(PythonPackage):
 
     maintainers("adamjstewart")
 
+    version("4.3.1", sha256="8b5f11c0e6a5308086b48e8f6841223a4fa1a37d513684612b7ee854b533015f")
     version("3.8.3", sha256="786adb0571f75114e9c5f7a31dd2e6eb39a9791f22c8757621545e2ded3ea367")
     version("3.8.1", sha256="33277fc0a8d7b0c7ce70fcc74bb82ad39f944c009b334856c6e86bf552b1dfdc")
     version("3.8.0", sha256="ec5de7ff2bfa8692fa96a846bb5aae52f267fc322fbbe303c1f042d258af5766")
 
     depends_on("python@2.7:2.8,3.5:", type=("build", "run"))
+    depends_on("python@3.8:", type=("build", "run"), when="@4.3.1:")
     depends_on("py-setuptools", type="build")
 
+    depends_on("py-cython@0.29.32:", type=("build", "run"), when="@4.3.1:")
```

Comment:
```suggestion
    depends_on("py-cython", type=("build", "run"), when="@4.3.1:")
```
Any version is supposed to work